supplier_app/constants/email_notifications.py

```diff
@@ -55,4 +55,15 @@
             'btn_url': '{}{}'.format(settings.BRITESU_BASE_URL, SUPPLIER_HOME_URL),
         },
     },
+    'taxpayer_in_progress': {
+        'subject': _('Your taxpayer is in progress now'),
+        'body': {
+            'upper_text': _(
+                'You Taxpayer is now in progress to Approve\n'
```

Comment:
Your taxpayer is now in progress to be approved*

Reply:
Done